app/services/protocol-links.ts

```diff
@@ -108,9 +116,11 @@ export class ProtocolLinksService extends Service {
     const appId = info.path.replace('/', '');
 
     if (this.platformAppsService.views.getApp(appId)) {
-      this.navigationService.navigateApp(appId);
+      this.navigationService.navigate('PlatformAppMainPage', { appId });
+      this.sideNavService.setCurrentMenuItem(appId);
```

Comment:
i might be misunderstanding but i thought `navigateApp` did these two things?

Reply:
Ah yes, it does. I was thinking about that optional key when setting the menu item at the time :sweat_smile: